x-pack/plugins/alerting/server/alert_type_registry.ts

```diff
@@ -40,7 +40,16 @@ export class AlertTypeRegistry {
         })
       );
     }
-    this.alertTypes.set(alertType.id, alertType);
+    // set default value for actionGroups if it is empty
+    if (alertType.actionGroups.length === 0) {
```

Comment:
It seems a bit odd to magically add to the array if the developer explicitly defined an empty array `[]`. I'm thinking we leave it to the developer to still define at least done action group?